README.md

```diff
@@ -86,6 +87,69 @@ deb(xx, yy, xxyy); // prints "xx, yy, xxyy = 3, 10, 103"
 ----------------
 
 
+## Generic Reader and Writer for multiple variables and containers
+```cpp
+template <typename... T>
+void read(T &...args) {
+    ((cin >> args), ...);
+}
+
+template <typename... T>
+void write(string delimiter, T &&...args) {
```

Comment:
I know its not trivial. But can we pass the delimeter at end and can it have a default too? Again, I know it might not be a straight arrow to pull this off. But can you try it out?

Reply:
sure, I can give it a try.

Reply:
I feel named parameters (C++20 feature) could be useful, however for now, I think using tuple as an intermediate solution should be good enough.
Also, for Competitive Programming, we need to be sure that this doesn't get too much slow. We can also leave it with current `void write(delimeter, ...)` signature as well otherwise.

Reply:
On another note, having multiple arguments and then one string default at end would really be ambiguous. 
write("x", "y") <-- is this printing x and y with default delimeter or y is the delimeter?

Lets use the current implementation itself then. 
Lets also rename writeContainer to take first argument as delimeter.

We will revisit this later. Lets close this for now.

Reply:
sure, I have updated the signature of `writeContainer` as well and fixed a minor typo.
Also, named parameters in C++20 is something that I wasn't aware of, I will read about it and probably create another PR if required.